docs/start/smeshing/smeshing_adv/poet.md

```diff
@@ -27,36 +27,50 @@ In the future, it is likely that instead of all PoET servers running in identica
+1. Update the configuration by adding the new PoET server to the list of existing servers (and remove any you do not
+   wish to use).
 1. Start the node again.
 
 When the next PoET round starts, the node will start using the new set of PoET servers as per the updated configuration.
 
 ### Switching Phase
 
-If you wish to switch to one or multiple PoET servers that operate on a different phase, then the default set of PoET servers ensures
-that your node and the PoET server(s) you are using have the following configuration parameters set to the _same values_:
+If you wish to switch to one or multiple PoET servers that operate on a different phase than the default set of PoET
+servers ensures that your node and the PoET server(s) you are using have the following configuration parameters set to
+the _same values_:
 
-- `"phase-shift"`: The time (relative to the beginning of an epoch) when a new PoET round starts. For the default
-  set of PoET servers, this value is **240 hours**. Lower values mean that the nodes starting to use the PoET server have to wait longer before
-  they become eligible to collect rewards, while higher values mean that nodes have less time after fetching a PoET to
-  publish their ATX to be eligible to collect rewards.
-- `"cycle-gap"`: The time between the end of a PoET round and the start of a new one. Larger values give nodes
-  more time to generate a PoST - since this has to happen within the cycle gap to not miss an epoch. Smaller values
-  give the PoET server more time to generate the PoET and collect more ticks. For the default set of PoET servers, this value is
-  **12 hours**.
+- `"phase-shift"`: The time (relative to the beginning of an epoch) when a new PoET round starts. For the default set of
+  PoET servers, this value is **240 hours**. Lower values mean that the nodes starting to use the PoET server have to
+  wait longer before they become eligible to collect rewards, while higher values mean that nodes have less time after
+  fetching a PoET to publish their ATX to be eligible to collect rewards.
+- `"cycle-gap"`: The time between the end of a PoET round and the start of a new one. Larger values give nodes more time
+  to generate a PoST - since this has to happen within the cycle gap to not miss an epoch. Smaller values give the PoET
+  server more time to generate the PoET and collect more ticks. For the default set of PoET servers, this value is **12
+  hours**.
 
-You should also ensure that any custom PoET has the same `"genesis-time"` and `"epoch-duration"` properties as the  mainnet
-(`"2023-07-14T08:00:00Z"` and 336 hours, respectively).
+You should also ensure that any custom PoET has the same `"genesis-time"` and `"epoch-duration"` properties as the
+mainnet (`"2023-07-14T08:00:00Z"` and 336 hours, respectively).
+
+**Note**: Switching to an earlier PoET phase than your node is currently using will result in your node missing one
```

Comment:
Do we want to mention the "bridge poet" ?

Reply:
Wdym by bridge PoET?

Reply:
Added